.github/.gitignore

```diff
@@ -0,0 +1 @@
+*.html
```

Comment:
I typically avoid committing non-root .gitignore

Reply:
So do I! I didn't even notice this, it's `usethis::use_github_action()`'s doing. Do you think we should revert it?